docs/menu.yml

```diff
@@ -80,7 +78,7 @@ catalog:
           - name: "Setup External Communication Channels"
             path: "/en/setup/backend/backend-expose"
           - name: "Kafka Fetcher"
-            path: "/en/setup/backend/Kafka-fetcher"
+            path: "/en/setup/backend/kafka-fetcher"
```

Comment:
This is a fix. But you should know actually this isn't an issue as website building fixes this automatically.

Check this menu https://skywalking.apache.org/docs/main/next/en/setup/backend/kafka-fetcher/

Reply:
Yes, I was mainly intended to fix the actual broken one for protocol, along with small fxies.

Reply:
@wankai123 told me this was broken. I am thinking how to write a new one, or actually we leave this part to API docs only.